include/OpenSpaceToolkit/Mathematics/Object/Interval.hpp

```diff
@@ -331,6 +331,125 @@ class Interval : public IntervalBase
+    ///                     };
+    ///                     Array<Interval<Real>> sorted = Interval<Real>::Sort(unsorted); // {[-3.0, 0.0], [-1.0, 1.0]}
+    /// @endcode
+    ///
+    /// @return             Sorted array
+
+    static ctnr::Array<Interval<T>> Sort(
+        const ctnr::Array<Interval<T>>& anIntervalArray, const bool& byLowerBound = true, const bool& ascending = true
+    );
+
+    /// @brief              Creates a new array without any overlaps and sorted by their lower bound
+    ///
+    /// @code
+    ///                     Array<Interval<Real>> array = {
+    ///                         Interval<Real>::Closed(-1.0, 1.0),
+    ///                         Interval<Real>::Closed(-2.0, 0.0),
+    ///                         Interval<Real>::Closed(3.0, 4.0),
+    ///                     };
+    ///                     Array<Interval<Real>> merged = Interval<Real>::Merge(array); // {[-2.0, 1.0], [3.0, 4.0]}
+    /// @endcode
+    ///
+    /// @return             Merged array
+
+    static ctnr::Array<Interval<T>> Merge(const ctnr::Array<Interval<T>>& anIntervalArray);
+
+    /// @brief              Creates a new array that contains the gaps between an array of intervals. If
+    ///                     an interval is actually defined, only intervals intersecting with the interval are
+    ///                     considered and potential leading and trailing gaps w.r.t. the interval boundaries are
+    ///                     included. The intervals are previously merged and sorted.
+    /// @code
+    ///                     Array<Interval<Real>> intervals = {
+    ///                         Interval<Real>::Closed(-1.0, 1.0),
+    ///                         Interval<Real>::Closed(2.0, 4.0),
+    ///                     };
+    ///                     const Interval<Real> bounds = Interval<Real>::Closed(1.5, 4.5)
+    ///                     Array<Interval<Real>> gaps = Interval<Real>::GetGaps(intervals, bounds);
+    ///                     // {[1.5, 2.0], [4.0, 4.5]}
+    /// @endcode
+    ///
+    /// @return             Gaps array
+
+    static ctnr::Array<Interval<T>> GetGaps(
+        const ctnr::Array<Interval<T>>& anIntervalArray, const Interval<T>& anInterval = Interval::Undefined()
+    );
+
+    /// @brief              Creates a new array that contains the intervals representing a logical 'OR' conjunction
+    ///                     between the two arrays.
+    /// @code
+    ///                     Array<Interval<Real>> anIntervalArray = {
+    ///                         Interval<Real>::Closed(-1.0, 1.0),
+    ///                         Interval<Real>::Closed(2.0, 4.0),
+    ///                     };
+    ///
+    ///                     Array<Interval<Real>> anotherIntervalArray = {
+    ///                         Interval<Real>::Open(0.5, 1.5),
+    ///                         Interval<Real>::Open(3.0, 5.0),
+    ///                         Interval<Real>::Open(7.0, 8.0),
+    ///                     };
```

Comment:
Can you add a line that explains how it basically performs a merge after combining the arrays. It's not obvious from the description what this does to me.